styles/styles.css

```diff
@@ -0,0 +1,209 @@
+body{
+    margin:0 0 0 0;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.pageheader{
+    background-color:#2c9ab7;
+    color:#ffffff;
+    background-image: url(../images/header-image.jpg);
+    background-size: cover;
+    width:100%;
+    height:300px;
+}
+
+.headertext{
+    padding-top: 60px;
+    padding-left: 60px;
+}
+
+.ulnav{
```

Comment:
this is kind of an award class name. What I would do is target the nav and then the ul
```
nav ul {
  // code
}
```